README.md

```diff
@@ -0,0 +1,92 @@
+# Infection Static Analysis Plugin
+
+This plugin is designed to run static analysis on top of [`infection/infection`](https://github.com/infection/infection)
+test runs in order to discover if [escaped mutants](https://en.wikipedia.org/wiki/Mutation_testing)
+are valid mutations, or if they do not respect the type signature of your
+program.
```

Comment:
```suggestion
program. If the mutation would result a type error, it is "killed".

TL;DR:

 - This will improves your mutation score, since mutations which result in type errors become killed.
 - This is very hacky, and replaces `vendor/bin/infection` essentially. Please read the `Stability` section below first for details.
 - This is currently much slower than running infection by itself. There are ideas/suggestions to improve this in the future.
```

Reply:
Is it "would result **in** a type error"? Asking because I never know whether to add `in`

Reply:
b7b501460592ecacbce781c8dc0e383917dd06fa